README.md

```diff
@@ -1 +1,30 @@
-# Portifolio
\ No newline at end of file
+# Portifolio
+
+> Mobile version skeleton.
+
+In this project I designed a mobile version for template 1 from figma.
+
+## Built With
+
+- HTML & CSS
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@NatiG25](https://github.com/NatiG25)
+- Twitter: [@NatiG87702270](https://twitter.com/NatiG87702270)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] This link is broken. Kindly fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.